release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -444,6 +444,44 @@ For more information, see the following sections:
+
+For more information, see xref:..//storage/persistent_storage/persistent_storage_local/persistent-storage-local.adoc#local-volume-cr_persistent-storage-local[Provisioning local volumes by using the Local Storage Operator].
+
+[id="ocp-4.15-storage-support-for-wiping-the-devices"]
+==== Support for wiping the devices
+
+This feature provides a new optional field `forceWipeDevicesAndDestroyAllData` in the `LVMCluster` custom resource (CR) to force wipe the selected devices. Prior to this release, wiping the devices required manually accessing the host. With this release, you can force wipe the disks without manual intervention. This simplifies the process of wiping the disks.
+[WARNING]
+====
+If `forceWipeDevicesAndDestroyAllData` is set to `true`, {lvms} wipes all previous data on the devices. You must use this feature with caution.
+====
+
+For more information, see xref:..//storage/persistent_storage/persistent_storage_local/persistent-storage-local.adoc#local-volume-cr_persistent-storage-local[Provisioning local volumes by using the Local Storage Operator].
+
+[id="ocp-4.15-storage-support-for-lvms-on-multi-node-clusters"]
+==== Support for deploying {lvms} on multi-node clusters
+
+This feature provides support for {lvms} on multi-node clusters. Prior to this release, {lvms} only supported single-node configurations. With this release, {lvms} supports all the {product-title} deployment topologies. This enables provisioning of local dynamic storage on multi-node clusters.
+[WARNING]
+====
+{lvms} only supports node local storage on multi-node clusters. It does not support storage data replication mechanisim across nodes. When you are using {lvms} on multi-node clusters, you must ensure storage data replication through active or passive replication mechanism to avoid single point of failure.
+==== 
+
+For more information, see xref:..//storage/persistent_storage/persistent_storage_local/persistent-storage-local.adoc#local-volume-cr_persistent-storage-local[Provisioning local volumes by using the Local Storage Operator].
+
+[id="ocp-4.15-storage-support-for-integrating-raid-arrays-with-lvms"]
+==== Integrating RAID arrays with {lvms}
+
+This feature provides support for integrating RAID arrays that are created using the `mdadm` utility with {lvms}.  The `LVMCluster` custom resource (CR) provides support for adding paths to the RAID arrays in the `deviceSelector.paths` field and the `deviceSelector.optionalPaths` field.
+
+[id="ocp-4.14-storage-fips-conpliance-support-for-lvms"]
+==== FIPS compliance support for {lvms}
+
+With this release, {lvms} is Federal Information Processing Standards (FIPS) compliant. When {lvms} is installed on {product-title} in FIPS mode, {lvms} uses the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-3 validation. 
```

Comment:
```suggestion
With this release, {lvms} is designed for Federal Information Processing Standards (FIPS). When {lvms} is installed on {product-title} in FIPS mode, {lvms} uses the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-3 validation. 
```

Reply:
Latest changes since 9.3 is not yet compliant (no NIST approval)

Reply:
You mean RHEL 9.3 is not yet FIPS compliant?

Could you provide more context?

Reply:
Yes of course:
RHEL 9.3 crypto libraries such as OpenSSL have been submitted to NIST for FIPS 140-3 validation. However they are not yet approved so technically they are not certified yet.

Reply:
Okay.
Is the last part of the line relevant to the users?
Can we rather say "When {lvms} is installed on {product-title} in FIPS mode, {lvms} uses the RHEL cryptographic libraries to "do something".

And add a note saying,
"Currently, RHEL 9.2 and lower versions are FIPS compliant. RHEL 9.3 is not yet FILPS compliant. 
RHEL 9.3 cryptographic libraries that have been submitted to NIST for FIPS 140-3 validation.(Optional)"

Reply:
We could write "When {lvms} is installed on {product-title} in FIPS mode, {lvms} uses the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-3 validation"

Reply:
And RHEL9.2 is also not compliant as I know: https://access.redhat.com/articles/2918071

Reply:
Which versions of RHEL are currently FIPS compliant?

Reply:
RHEL cannot be FIPS compliant only the underlying crypto libraries like openssl. And solutions like LVMS can only be "Designed for FIPS" and not be FIPS compliant themselves. Maybe reach out to @DanielFroehlich for specifics on the communication?

Reply:
Hi @DanielFroehlich 

Is the following technically correct?
"With this release, {lvms} is designed for Federal Information Processing Standards (FIPS). When {lvms} is installed on {product-title} in FIPS mode, {lvms} uses the RHEL cryptographic libraries that are FIPS compliant."

From the release note perspective, is there any additional info that we should add here?

Reply:
Thanks @DanielFroehlich for confirming that "we should avoid talking about version and their actual fips status".